### ISSUE TITLE
Properties not passed to child classes of EnvObject

### ISSUE THREAD

**Issue by jwaa** [matrx-software/matrx#173]
**Describe the bug**
When you use an child class of `EnvObject`, such as `SquareBlock` any mandatory properties such as  `visualize_colour` are not taken into account if this is not an explicit parameter in the constructor.

**To Reproduce**
The code snippet below should result in a 3x3 world with a black `SquareBlock` in the middle, instead it is a light-blue `SquareBlock` as the `visualize_colour` property is passed in the `**custom_properties` argument of `SquareBlock.__init__` which is ignored in the `WorldBuilder.__create_env_object` method.

```python
builder = WorldBuilder(shape=(3, 3), run_matrx_api=True, run_matrx_visualizer=True, visualization_bg_clr="#f0f0f0", visualization_bg_img="")
builder.add_object((1,1), "Block", callable_class=SquareBlock, visualize_colour="#000000")
builder.startup()
world = builder.get_world()
world.run(builder.api_info)
builder.stop()
```

**Expected behavior**
That any mandatory property that is passed to the constructor's **kwargs should be recognized and added to the object! 

**Screenshots**
N/A

**Stacktrace**
N/A

**Additional context**
It may prove that the bug is even larger than just ignoring the mandatory properties, it could be that *all* **kwargs properties are ignored. This should be tested for.

**Comment by jwaa** [comment]
A work around for now is to add the required (mandatory) property in the constructor of the child's class or to pass a default value of it to the `EnvObject`'s super constructor.

We need to think about what the usage is of passing a keyword argument in **kwargs to a child's class of `EnvObject` when this argument is a mandatory property. Does the fact that the property was not mentioned explictely in the constructor that the user intended it to do nothing (making this bug actually the desired behaviour)?

**Comment by thaije** [comment]
Duplicate with and fixed in #281
